components/camel-sjms/src/main/java/org/apache/camel/component/sjms/SjmsEndpoint.java

```diff
@@ -98,6 +99,9 @@
     private boolean persistent = true;
     @UriParam(label = "consumer")
     private String durableSubscriptionId;
+    private String subscriptionId;
```

Comment:
You need to add UriParam for these new options as well. For the new options that are on for jms2 you can move to it instead of keeping it here on the old component